runners/direct-java/build.gradle

```diff
@@ -64,6 +64,8 @@ dependencies {
   shadow library.java.slf4j_api
```

Comment:
It seems like our list of shaded projects doesn't match the list defined up above, looks like we are missing `:beam-runners-java-fn-execution`, `:beam-sdks-java-fn-execution`. This seems worthwhile to fix regardless of the bytebuddy/commons_lang3

Reply:
But we list java-sdks-core as shadow dependency, which means we don't want to include this into our project right?

Reply:
shadow means its not shaded, so its dependencies should automatically become dependencies of this project.

Reply:
I checked beam-runners-direct-java shadowJar, it doesn't include shaded byte_buddy from sdks-java-core

Reply:
Fixed shaded projects list: https://github.com/apache/beam/pull/5467/commits/0354ebd43bcd10a90f9369596e275010cc51e6ff

Reply:
Fixing the shaded project list is good but I still think adding library.java.byte_buddy and library.java.commons_lang3 here is incorrect.